PWGHF/Utils/utilsEvSelHf.h

```diff
@@ -197,6 +207,10 @@ struct HfEventSelection : o2::framework::ConfigurableGroup {
     }
   }
 
+  void initRctChecker() {
```

Comment:
Maybe we can take the opportunity to define a common `init` function and move the Zorro initialisation and `addHistograms` inside.
```suggestion
  void init(o2::framework::HistogramRegistry& registry) {
```

Reply:
Done